docs/source/references/modules.rst

```diff
@@ -543,6 +543,11 @@ WER
 .. autoclass:: torchmetrics.WER
     :noindex:
 
+BERTScore
```

Comment:
could you move this to be the first metric in the text section (alphabetical order)?